README.md

```diff
@@ -1,4 +1,5 @@
-# [![Build Status](https://travis-ci.org/aces/Loris.svg?branch=master)](https://travis-ci.org/aces/Loris) [![Documentation Status](https://readthedocs.org/projects/acesloris/badge/?version=latest)](https://acesloris.readthedocs.io/en/latest/?badge=latest)
+![](./loris-logo.svg)
```

Comment:
Love the change, the logo is currently sized extra.  How about re-sizing it using this syntax below?  (no idea if 80 is the right value, might need tinkering)

```suggestion
![](./loris-logo.svg | width=80)
```